administrator/language/de-DE/com_guidedtours.ini

```diff
@@ -50,6 +52,8 @@ COM_GUIDEDTOURS_STEP_FILTER_SEARCH_LABEL="Schritte durchsuchen"
 COM_GUIDEDTOURS_STEP_NEW_STEP="Neuer Schritt"
 COM_GUIDEDTOURS_STEP_POSITION_DESC="Position des Schritt-Popups, relativ zu dem Element auf das es zeigt."
 COM_GUIDEDTOURS_STEP_POSITION_LABEL="Position"
+COM_GUIDEDTOURS_STEP_TARGETNOTE_MESSAGE="Wenn ein Schritt als interaktiv gekennzeichnet ist, können bestimmte interaktive Optionen zusätzliche Parameter für die Interaktion des Benutzers mit einem Zielelement enthalten."
```

Comment:
Dieser Satz ist irgendwie kompliziert und verstehe in auch im Englischen nicht wirklich 😅.
Könnte mir hier so vorstellen, das vom Sinn her so etwas in die Richtung gemeint ist: Auch wenn es sich um ein interaktiver Schritt handelt, hat der Benutzer dennoch die vorgegebene "Sache" zu machen, um das vorgegebene Ziel zu erreichen (weil sonst die Tour auch nicht funktionieren würde). 🤔

Reply:
Ich würde das jetzt erstmal mit Zielwert nehmen.